algorithm/WebRelated/WordsAndCharsCounter/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Count total words and characters</title>
+</head>
+
+<body>
+    <textarea name="strings" id="strings" cols="50" rows="20" style="font-size: 20px;"></textarea>
+    <h2 id="totalOfWords"></h2>
+    <h2 id="totalOfChars"></h2>
+</body>
+
+<script>
+    const strings = document.getElementById("strings");
+    const totalOfWords = document.getElementById("totalOfWords");
+    const totalOfChars = document.getElementById("totalOfChars");
+
+    strings.addEventListener("keyup", function() {
+        totalOfWords.textContent = `Total of words : ${strings.value.split(" ").filter(string => string !== " ").length}`;
+        totalOfChars.textContent = `Total of characters : ${strings.value.length}`;
+    });
+</script>
```

Comment:
Pake file js yang udah ada aja

```suggestion
<script src="counter.js"></script>
```